CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+# #develop#
+
+## Breaking changes
+
+## New deprecations
+
+* Deprecated `MDCAppBarTextColorAccessibilityMutator`.
+
+### FlexibleHeader
+
+* [minMaxHeightIncludesSafeArea](https://github.com/material-components/material-components-ios/issues/4764)
+
+## New features
```

Comment:
Delete this section if there are no new features.

Reply:
added something